deploy/lib/control/CasinoController.php

```
@@ -20,7 +20,12 @@ class CasinoController {
 	 * @return Array
 	 */
 	public function index() {
-		$player = new Player(self_char_id());
+		$player = Player::find(self_char_id());
+
+        if (!$player) {
+            $player = new Player();
```

Hmmm, shouldn't it then ideally be something like:

if(self_char_id()){
    $player ::find(self_char_id());
} else {
   $player = new Player();
}

'cause we never want there to be an existent logged in player, who can't get their player object.  If that occurs, that seems like it shouldn't overwrite with a new Player(), it should just fail fast.